core/platform/datastore/gae_datastore_services_test.py

```diff
@@ -99,6 +99,9 @@ class MockDatetimeForDatastoreTests(test_utils.GenericTestBase):
     def test_exception_is_raised_when_passed_wrong_type(self):
         # type: () -> None
         with self.assertRaisesRegexp(Exception, 'mocked_now must be datetime'): # type: ignore[no-untyped-call]
+            # TODO(#13528): Remove this test after back-end is fully type
```

Comment:
Here and elsewhere: I think we typically use "backend"/"frontend" and don't hyphenate, when talking about the noun. Could you please fix this everywhere? Probably clearer to say "the backend" instead of "back-end".

Also it should be "type-annotated".

Reply:
Done